applets/notification_area/main.c

```diff
@@ -214,6 +245,9 @@ na_tray_applet_realize (GtkWidget *widget)
                                               action_group);
   g_object_unref(action_group);
 
+  setup_gsettings (applet);
+  load_gsettings (applet);
```

Comment:
`load_gsettings` seems unneeded / duplicated code. You can call `gsettings_changed_min_icon_size (applet->priv->settings, KEY_MIN_ICON_SIZE, applet);` to load initial settings.

Reply:
ok, I will change that